backend/src/main/java/com/carffeine/carffeine/station/service/review/ReplyQueryService.java

```diff
@@ -0,0 +1,29 @@
+package com.carffeine.carffeine.station.service.review;
+
+import com.carffeine.carffeine.station.infrastructure.repository.review.ReplyQueryRepository;
+import com.carffeine.carffeine.station.infrastructure.repository.review.dto.ReplyResponse;
+import com.carffeine.carffeine.station.infrastructure.repository.review.dto.ReplyResponses;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+@Service
+public class ReplyQueryService {
+
+    public static final int NEXT_PAGE_INDEX = 1;
+    public static final int NO_MORE_PAGE = -1;
+    private final ReplyQueryRepository replyQueryRepository;
+
+    public ReplyResponses findAllReplies(Long reviewId, Pageable pageable) {
+        Page<ReplyResponse> allReplies = replyQueryRepository.findAllReplies(reviewId, pageable);
+        return new ReplyResponses(allReplies.getContent(), getNextPage(pageable, allReplies));
+    }
+
+    private static int getNextPage(Pageable pageable, Page<ReplyResponse> allReplies) {
+        return allReplies.hasNext() ? pageable.getPageNumber() + NEXT_PAGE_INDEX : NO_MORE_PAGE;
+    }
```

Comment:
첫번째 파라미터가 pageable이여야할 이유가 있을까요? 저는 그냥 숫자만 받아오는게 좋을 것 같아요
```suggestion
    private static int getNextPage(int pageNumber, Page<ReplyResponse> allReplies) {
        return allReplies.hasNext() ? pageNumber + NEXT_PAGE_INDEX : NO_MORE_PAGE;
    }
```

Reply:
좋네요. 단축키로 메서드 뽑다보니 이부분은 신경쓰지 못한 것 같아요. 감사합니다